app/assets/javascripts/modules/external_users/claims/SideBar.js

```diff
@@ -49,11 +49,11 @@ moj.Modules.SideBar = {
     var self = this;
 
     this.totals = {
```

Comment:
@cleargif could you review this JS changes and let me know if you're happy with them?